s3_sequins_test.go

```diff
@@ -21,21 +22,20 @@ import (
 
 type S3ClientMock struct {
 	s3iface.S3API
-	toggle             bool
+	counter            int
```

Comment:
Had to change the way this test worked since `GetObjectOutput` is now called more frequently due to checks for the `_SYMLINK` file.